packages/story-editor/src/components/library/panes/media/common/mediaGallery.js

```diff
@@ -52,21 +53,8 @@ function MediaGallery({ resources, onInsert, providerType, canEditMedia }) {
     height: resource.height,
   }));
 
-  const containerRenderer = ({ children, containerProps }, ref) => {
-    return (
-      <div
-        ref={ref}
-        {...containerProps}
-        style={{
-          ...containerProps.style,
-          flexDirection: 'row',
-          flexWrap: 'wrap',
-          justifyContent: 'space-evenly',
-        }}
-      >
-        {children}
-      </div>
-    );
+  const containerRenderer = ({ children }, ref) => {
+    return <GalleryContainer ref={ref}>{children}</GalleryContainer>;
```

Comment:
Is `containerProps.style` needed here?

Reply:
`containerProps.style` has 4 properties, 3 needed to overridden ( except `display` ).